content/source/docs/cloud/workspaces/state.html.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: "cloud"
+page_title: "Terraform State - Workspaces - Terraform Cloud"
+---
+
+# Terraform State in Terraform Cloud
+
+-> **API:** See the [State Versions API](../api/state-versions.html).
+
+Each Terraform Cloud workspace has its own separate state data, used for runs within that workspace.
+
+In [remote runs](../run/index.html), Terraform Cloud automatically configures Terraform to use the workspace's state; the Terraform configuration does not need an explicit backend configuration. (If a backend configuration is present, it will be overridden.) In local runs, you can use a workspace's state by configuring [the `remote` backend](/docs/backends/types/remote.html).
+
+## State Versions
+
+In addition to the current state, Terraform Cloud retains historical _state versions,_ which can be used to analyze infrastructure changes over time.
```

Comment:
```suggestion
In addition to the current state, Terraform Cloud retains historical state versions, which can be used to analyze infrastructure changes over time.
```

Reply:
Yeah, I guess that doesn't need to be introduced as a vocab term. My own version of Gravitas Caps. 😰